Formula/google-protobuf.rb

```diff
@@ -59,6 +65,7 @@ def install
     ENV.universal_binary if build.universal?
     ENV.cxx11 if build.cxx11?
 
+    (buildpath/"gmock").install resource("gmock")
```

Comment:
@stanley-cheung I have no idea how exactly the `.install` comes from, but the trigger of unzip action comes from [the Resource class](https://github.com/Homebrew/brew/blob/4b2c4ef25835a87b5bac4fb26eae91dbfed93863/Library/Homebrew/resource.rb#L93). And, there is [another `.install` call like this](https://github.com/Homebrew/brew/blob/4b2c4ef25835a87b5bac4fb26eae91dbfed93863/Library/Homebrew/resource.rb#L101).